docs/source/delta-managed-commit.md

```diff
@@ -0,0 +1,88 @@
+---
+description: Learn about Delta Managed Commits.
+---
+
+# Delta Managed Commits
+
+.. warning:: This feature is available in preview in <Delta> 4.0. Since this feature is still in preview, it can undergo major changes. Furthermore, a table which has this preview feature enabled cannot be written to by future Delta releases until
+this feature is manually dropped from the table.
+
+The [Managed Commit](https://github.com/delta-io/delta/issues/2598) writer table feature defines a general protocol which can be extended to define custom commit protocols. The feature enables clients to designate "commit owners" for tables, which means that every commit to the table must be managed by that commit owner.
+The presence of a commit owner which is responsible for all commits to a table makes multi-cluster and multi-engine writes to tables safe. The commit owner for a table can be discovered by reading the table through the filesystem.
+
+
```

Comment:
Should we add a compatibility section?

  ## Compatibility
  Managed Commit is a writer table feature.
  - That means only clients which understands this feature could write to a managed commit table. 
  - Older clients which do not understand this table feature could still read a managed commit table. Note that the read may give stale results depending on [table's commit owner backfill policy](#hyperLinkToBackfillSectionInProtocolRFC.md).
  ```